src/witan/models/dem/ccm/mort/mortality_mvp.clj

```diff
@@ -8,6 +8,10 @@
             [incanter.core :as i]))
 
 (defn create-popn-at-risk
+  "Takes datasets with population, deaths, and births. Returns a new
```

Comment:
add to beginning: "Creates population at risk for calculating historic age specific mortality rates."